packages/fxa-settings/src/styles/brand-banner.css

```diff
@@ -0,0 +1,53 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+.brand-banner-bg {
+  background: linear-gradient(
+    88.76deg,
+    #e4eaf6 3.37%,
+    #dbeef8 39.93%,
+    #daf3f4 65.09%,
+    #e3f6ed 102.21%
+  );
+}
+
+.m-logo {
+  width: 30px;
```

Comment:
Are we planning a follow up to use Tailwind? If so that's fine, but I would push back on design decisions that get us outside of the existing design system if not. At the least, we should use `rem` here instead of `px` so it scales up/down with everything else. We can also place shared CSS between content-server and settings in `fxa-react` (like with our button classes) - totally custom CSS like this should only be done for one-offs. Granted I haven't checked out the branch though and am commenting from just looking at the code.

Reply:
Also related (Component classes section): https://github.com/mozilla/fxa/blob/ae13d506bd1119374d5b5d0dae422bd31c818faf/packages/fxa-content-server/app/styles/tailwind/README.md#L17

Reply:
Yeah, one nice thing about the design guide in Storybook is it can be used as quick reference to convert px to rem since that's what Figma gives us (though the IDE will now show you the pixel equivalent to some Tailwind classes which is super nice). We can get away with stuff like `2.25rem` etc. to get more exact pixel value equivalents but if there's any weird oddly specific values sometimes it's a mistake in Figma because something saved weird, or Design is more than happy to go with a very close value that matches the system we've got. Stuff like gradients, we can either make one-offs for like what you have here, or add it to the Tailwind config for later use. I feel like this is fine as a 1-pointer follow up or if this banner is going to go away by end of year then maybe it's also fine to just ignore me. 😛